.github/workflows/ci.yml

```
@@ -249,27 +249,21 @@ jobs:
 
       - name: Build docs
         run: target/debug/xtask doc --deny-warnings
+      
+      - name: Compress docs
+        if: github.event_name == 'pull_request'
+        run: |
+          cd target
+          tar --zstd -cf docs.tar.zstd doc
+          cd ..
+          mv target/docs.tar.zstd .
```

This should be equivalent:

```suggestion
          tar --zstd -cf docs.tar.zstd -C target doc
```